apps/cli/src/commands/link.ts

```diff
@@ -0,0 +1,57 @@
+import { stdin as input, stdout as output, cwd } from "node:process";
+import { join } from "node:path";
+import * as readline from "node:readline/promises";
+import { readFile, writeFile } from "node:fs/promises";
+import {
+  GLOBAL_CONFIG_FILE,
+  LOCAL_CONFIG_FILE_NAME,
+  PROJECT_NAME,
+} from "../constants";
+import type { Command } from "../args";
+import { ensureFileInPath } from "../fs-utils";
+
+export const link: Command = async () => {
+  const rl = readline.createInterface({ input, output });
+  const shareLink = await rl.question(`Paste share link (with build access): `);
+
+  const shareLinkUrl = new URL(shareLink);
+  const host = shareLinkUrl.origin;
+  const token = shareLinkUrl.searchParams.get("authToken");
+  const paths = shareLinkUrl.pathname.split("/").filter(Boolean);
```

Comment:
Do you know what filter(boolean) for?

Reply:
No idea, i will re-check the logic then. Just copied it from before 👍